bentoml/cli/__init__.py

```diff
@@ -105,6 +105,16 @@ def info(archive_path=archive_path):
         )
         print(output)
 
+    # Example usage: bentoml docs /SAVED_ARCHIVE_PATH
+    @bentoml_cli.command(
+        help="Display API documents in Open API format", short_help="Display docs"
+    )
+    @conditional_argument(archive_path is None, "archive-path", type=click.STRING)
+    def docs(archive_path=archive_path):
+        model_service = load(archive_path)
+        server = BentoAPIServer(model_service)
```

Comment:
this could be just ```get_docs(bento_service)``` right? is there any reason we need to initialize BentoAPIServer here?

Reply:
Yep, that's easier. I'll export `get_docs` then.